module/VuFind/src/VuFind/Auth/ILS.php

```diff
@@ -113,6 +113,23 @@ public function authenticate($request)
         return $this->handleLogin($username, $password, $loginMethod, $rememberMe);
     }
 
+    /**
+     * Does this authentication method support password recovery
+     *
+     * @return bool
+     */
+    public function supportsPasswordRecovery()
```

Comment:
This needs a counterpart in MultiILS that supports checking the selected target. For inspiration: 

- https://github.com/NatLibFi/NDL-VuFind2/blob/dev/themes/finna2/templates/Auth/MultiILS/login.phtml#L64 
- https://github.com/NatLibFi/NDL-VuFind2/blob/dev/themes/finna2/js/finna-layout.js#L372
- https://github.com/NatLibFi/NDL-VuFind2/blob/dev/module/Finna/src/Finna/Auth/ILSFinna.php#L53

Reply:
My initial work is only supposed to be for the ILS auth. MultiILS is something I'll look at in the future but I'm not sure it's part of the scope of this work. Is this piece a necessity? If so, I can start working on this and I'll resubmit when I have it done. I only have one ILS and I can't really test but I'm happy to look into it.